contracts/src/bridge/IBridgeSubordinate.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity >=0.7.0 <0.9.0;
+
+// IBridgeSubordinate is the interface for the layer 2 bridge, which accepts commands from the L1 bridge.
+interface IBridgeSubordinate {
```

Comment:
That's how the interfaces end up using with this design pattern for cross chain calls, but I've never seen them being called RPCs, since technically you encode selectors and params rather than call directly as you would using an rpc client.

Reply:
we need to find a better name for it. 
Subordinate doesn't really give you any information